File: java core/dec-4-2020/1.java
interface I1
{
void m();
}

interface I2 
{
void n();
}

class Doo
{
void o()
{
System.out.println("Hello Doo");
}
}
class Foo extends Doo implements I1,I2 
{
public void m()
{
System.out.println("Hello");
}
public void n()
{
System.out.println("welcome");
}
}

class Test
{
public static void main(String arg[])
{
Foo f=new Foo();
f.m();
f.n();
f.o();
}
}

File: cpp/.30 july/1.cpp
//pure virtual function
#include<iostream>
using namespace std;


class Foo
{
public:

void m()
{
cout<<"Hello all"<<endl;
}

virtual void n()=0;
};


class Doo:public Foo
{
public:
void n()
{
cout<<"welcome"<<endl;
}
};


int main()
{
Doo d;
d.m();
d.n();
}